src/debuginfo.cpp

```diff
@@ -734,7 +734,7 @@ static void get_function_name_and_base(const object::ObjectFile *object, bool in
         saddr = nullptr;
     // Try platform specific methods first since they are usually faster
     if (saddr && !*saddr) {
-#if defined(_OS_LINUX_) && !defined(JL_DISABLE_LIBUNWIND)
+#if (defined(_OS_LINUX_) || defined(_OS_FREEBSD_)) && !defined(JL_DISABLE_LIBUNWIND)
```

Comment:
Does this mean that the answer to the question I asked in https://github.com/JuliaLang/julia/issues/20798#issuecomment-283346606

> see if the libunwind on FreeBSD support that too

is yes?